lib/Form/Processor/ContentTypeFormProcessor.php

```diff
@@ -116,5 +133,20 @@ public function processRemoveContentTypeDraft(FormActionEvent $event)
                 new RedirectResponse($this->router->generate($this->options['redirectRouteAfterPublish']))
             );
         }
+
+        $this->addNotification('content_type.notification.draft_removed');
+    }
+
+    /**
+     * Pushes a flash notification to session.
+     *
+     * @param string $message
+     */
+    private function addNotification($message, array $params = [])
```

Comment:
Isn't this something we would move to a service somehow ? A `NotificationFlashBag` thing, that receives the session, and expects the notification string ?